components/bio-formats-tools/src/loci/formats/tools/ImageInfo.java

```diff
@@ -1144,19 +1144,31 @@ public static void main(String[] args) throws Exception {
         newArgs[idx] = scanner.nextLine();
         System.out.println("====% " + newArgs[idx]);
         try {
-        	new ImageInfo().testRead(newArgs);
+        	 new ImageInfo().testRead(newArgs);
```

Comment:
Stylistic, can these be updated to use an indentation of two spaces - https://bio-formats.readthedocs.io/en/latest/developers/code-formatting.html#java